src/backend/controllers/PostController.js

```diff
@@ -292,3 +292,21 @@ export const deletePostHandler = function (schema, request) {
     );
   }
 };
+
+/**
+ * This handler handles gets all posts in the db.
+ * send GET Request at /api/posts/page/pageNum
+ * */
+//=>0,1,2,3
+//1=>0,1,2,3,4,5,6,7
+//2=>0, 1,2,3,4,5,6,7,8,9,10,11
```

Comment:
remove these comments

Reply:
I have Kept this for future reference/explanation of pageNum